spec/models/shared_ex_scope_updated_in_date_range_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+# RSpec specification for a class with the 'updated_in_date_range' scope (class method)
+#
+# Pass in the method (symbol) that FactoryBot should use to create an instance of the class
+# Ex:
+#   To test the Payment class, assuming there is a factory method ':payment', you would say:
+#    it_behaves_like 'it_has_updated_in_date_range_scope', :payment
+#
+#   Note the Factory must have  updated_at as an attribute that can be set
+RSpec.shared_examples 'it_has_updated_in_date_range_scope' do | factory_method |
```

Comment:
it's not a place for shared examples, should be for example in support folder `support/shared`